File: POOJAVA/src/clasesPOO/Alumno.java
package clasesPOO;
class Alumno {
    String idAlumno;
    int IdPers;
    int IdCarrera;

    public Alumno(String idAl)
    {
        establecerDatos(idAl);
    }
    public String ObtenerIdAlumno()
    {
       return idAlumno;
    }
    void  establecerDatos(String idAlu)
    {
        idAlumno=idAlu;
    }
}

File: POOPHP/app/Model/Alumno.php
<?php
namespace App\Model;
class Alumno
{
   public $idAlumno="";
   public $IdPers="";
   public $IdCarrera="";
    public function __construct(String $idAl)
    {
        $this->establecerDatos($idAl);    
    }
    public function ObtenerIdAlumno()
    {
        return $this->idAlumno;
    }
    public function establecerDatos(String $idAlu)
    {
        $this->idAlumno=$idAlu;
    }
}
?>